calculator/lexer.py

```diff
@@ -1,4 +1,56 @@
-def lex(s):
+OPS = ["+", "-", "*", "/"]
+OPS_WITHOUT_SUBTRACTION = ["+", "*", "/"]
+
+
+def is_numeric_char(char):
+    """
+    is_numeric_char checks if a char is a number or a decimal point.
+    Input: char 
+    Output: True or False
+    """
```

Comment:
These docstrings aren't quite the right format.  Consider [this]( https://www.programiz.com/python-programming/docstrings) is an example:
```
      """
        Determines if a character is a numeric character (i.e. a character which may appear in a number).

        Parameters
        ----------
        char : char
            The character to be evaluated

        Returns
        -------
        bool: Whether or not this character is a numeric character.
        """
```
Please correct the docstrings in the other functions and add a proper one for `lex`